src/panels/lovelace/editor/config-elements/entity-selectors/hui-glance-entity-select.ts

```diff
@@ -0,0 +1,129 @@
+import { html, PropertyDeclarations } from "@polymer/lit-element";
+import { TemplateResult } from "lit-html";
+import "@polymer/paper-dropdown-menu/paper-dropdown-menu";
+import "@polymer/paper-item/paper-item";
+import "@polymer/paper-listbox/paper-listbox";
+
+import { HuiEntityEditor } from "../../../components/hui-entity-editor";
+import { HomeAssistant } from "../../../../../types";
+import { ConfigEntity } from "../../../cards/hui-glance-card";
+
+export class HuiGlanceEntitySelect extends HuiEntityEditor {
```

Comment:
Let's keep the names the same
```suggestion
export class HuiGlanceEntityEditor extends HuiEntityEditor {
```

Reply:
make sure the filename matches too